MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+# Maintainers
+
+|GitHub ID| Name | Email|
+|:---:| :----:| :---:|
+|[allencloud](https://github.com/allencloud)|Allen Sun|shlallen1990@gmail.com|
+|[skoo87](https://github.com/skoo87)|Marcky Wu|marckywu@gmail.com|
+|[Letty5411](https://github.com/Letty5411)|Letty Liu|letty.ll@alibaba-inc.com|
+|[rudyfly](https://github.com/rudyfly)|Ruby Zhang|rudyflyzhang@gmail.com| 
```

Comment:
My English name is Rudy Zhang，thank you. T_T

Reply:
Good catch and sorry for my carelessness. Updated. PTAL